sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -393,6 +393,12 @@ public:
 
   void make_edge(node& src, node& dest);
 
+  node add_barrier(const std::vector<event> eventList = {});
+
+  node make_external_event(event& recordedEvent, event& externalEvent, bool updatable = false);
+
+  void update_external_event(node externalEventNode, event& newExternalEvent);
```

Comment:
the preconditions of this function say " This member function is only available when the `command_graph` state is
  `graph_state::executable`", so should be under `command_graph<graph_state::executable>`